packages/theming/atlas/src/themesource/atlas_core/native/core/widgets/radiobuttons.ts

```diff
@@ -0,0 +1,54 @@
+import { border, brand, font, spacing } from "../../variables";
+import { RadioButtonsStyles } from "../../types/widgets";
+/*
+
+DISCLAIMER:
+Do not change this file because it is core styling.
+Customizing core files will make updating Atlas much more difficult in the future.
+To customize any core styling, copy the part you want to customize to styles/native/app/ so the core styling is overwritten.
+
+==========================================================================
+    RadioButtons
+
+    Default Class For Mendix RadioButtons Widget
+========================================================================== */
+
+const RADIO_BUTTON_OUTER_CIRCLE_SIZE = 16;
+const RADIO_BUTTON_INNER_CIRCLE_SIZE = 8;
+
+export const com_mendix_widget_native_radiobuttons_RadioButtons: RadioButtonsStyles = {
+    labelTextStyle: {
+        color: font.colorTitle,
```

Comment:
It's good practice to comment which properties are allowed here as this gets transpiled to javascript in `MX_PROJECT/themesource`. See this as an [example](https://github.com/mendix/widgets-resources/blob/master/packages/theming/atlas/src/themesource/atlas_core/native/core/widgets/switch.ts#L19) 

Also, it is helpful for end-users if we give them an easy way to customise widgets using their `MX_PROJECT/theme` by providing a "variables" object like [this](https://github.com/mendix/widgets-resources/blob/master/packages/theming/atlas/src/theme/native/custom-variables.ts#L509). 

These values above overwrite [these defined in `themesource`](https://github.com/mendix/widgets-resources/blob/master/packages/theming/atlas/src/themesource/atlas_core/native/variables.ts#L525), the combined result is used in the [widget styling in themesource](https://github.com/mendix/widgets-resources/blob/master/packages/theming/atlas/src/themesource/atlas_core/native/core/widgets/introscreen.ts#L25). 

Note: Atlas's themesource is distributed independently of `theme`. 

^^ 

@UrazAkgultan @Ahmad-Elsayed

Reply:
btw, if you dl the [Atlas_Core module from the marketplace](https://marketplace.mendix.com/link/component/117187), change extension to .zip then inspect, you can get an idea of what in includes from widget-resources